docs/troubleshoots.md

```diff
@@ -0,0 +1,28 @@
+# Troubleshoots 
+
+## Setting up Gmail to be used as default mail server in .env
+
+Gmail required applications that will use their service to be authorized first beforehand. Failure in doing this will cause SMTP sending mail process fail with the response **"534-5.7.9 Application-specific password required..."**.
+
+To solve this issue, you need to give this application an App password that is generated from Gmail account.
+
+Here are the steps:
+
+1. Login to your Gmail account and click on **Security** tab from the top navbar.
```

Comment:
Would it be possible to add a screenshot of how to get to the security tab?

Reply:
@mtreacy002 I really like this suggestion from @rpattath

Reply:
got it. will add it now